### DISCUSSION TITLE
Use Classic Scoring for Total Score on submission

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#15710] [Ideas] [answered]
As of right now, when submitting a play on lazer, the amount of score added to your Total Score on your profile is using the Standardised Scoring. This is a problem, because it makes leveling up practically impossible as the amounts needed are much higher.

My suggestion would be to use the Classic Scoring for the Total Score statistic as the values are closer to stable and would work much better with the leveling system.

**Comment by peppy** [comment, marked as answer]
All scores are going to be recalculated. This is out of scope.

**Reply by respektive** [reply]
i think you misunderstood, i was talking about total score which can not be recalculated as its the total of all passed and failed attempts. Unless you are planning to reset total score completely once lazer takes fully over.

**Reply by peppy** [reply]
We may rebalance the level system at some point, but the total score number isn't supposed to be something of great use itself. It's a fluff stat and the rate of increase shouldn't matter for any real purpose.

**Comment by WitchOfFrost** [comment]
There is an additional issue with this which came into perspective when thinking/talking about Total Score using Standardised Scoring in relation to the future of Ranked Score (Rankings).

At some point, there will be players who have a bigger/higher Ranked Score than Total Score if the idea proposed in one community meeting (https://www.youtube.com/watch?v=d66pU5lsHvE&t=1216s) is passed as-is.

This may end up confusing or entirely misleading.

**Reply by peppy** [reply]
Not sure I follow. The same number would be used for both.

**Reply by WitchOfFrost** [reply]
Alright, let me clarify.

- Ranked Score is the total value of all the highest scores on each ranked map, this means that only the best score of each map contributes to that value.
- Total Score is the absolute total of all scores submitted (used to be all ranked maps, but this was expanded to all maps as of early 2021 if i remember the date correctly). Every score submitted contributes to this.

My current understanding is that Ranked Score would follow the "classic" scoring system in its updated implementation by respektive. Which means the score contributed to it per play could exceed 1 million.

However, Total Score will follow the "Standardized" System, capping each plays score potential to 1 million for a nomod score.

This would, after some time and especially looking at new/medium term players cause Ranked Score to become higher than Total Score. This would not make sense since Total Score would be a total of all scores while ranked score would only be a subset of it.

Total Score and Ranked Score using the "same number" is not true in this sense. Unless you mean the same "scoring mode"?

**Reply by peppy** [reply]
If we scaled existing ranked score values, we would scale total score by the same amount. As such, ranked score could not exceed total score for any user, one would hope.
